src/database/migrations/2022_11_15_143153_duplicates.php

```diff
@@ -20,6 +20,7 @@ function (Blueprint $table) {
                 $table->foreignUuid("person2_id");
                 $table->foreignUuid("crew_id");
                 $table->foreignUuid("command_run_id")->nullable();
+                $table->foreignUuid("matching_algorithm_id")->nullable();
```

Comment:
Did you create a table with the name matching algorithm ?

If not I don't get the goal of this. 

Also please create another migration file to add this column instead of adding it directly into this migration file.It forces to reset the db, which isn't what we want.